core/src/main/java/nu/yona/server/analysis/service/ActivityService.java

```diff
@@ -167,7 +165,7 @@ private void createAndSaveInactivityDays(UserAnonymizedDTO userAnonymized, Set<G
 	private long getTotalPageableItems(UserAnonymizedDTO userAnonymized, ChronoUnit timeUnit)
 	{
 		long activityMemoryDays = yonaProperties.getAnalysisService().getActivityMemory().toDays();
-		Optional<ZonedDateTime> oldestGoalCreationTime = userAnonymized.getOldestGoalCreationTime();
+		Optional<LocalDateTime> oldestGoalCreationTime = userAnonymized.getOldestGoalCreationTime();
 		long activityRecordedDays = oldestGoalCreationTime.isPresent() ? (Duration
 				.between(oldestGoalCreationTime.get(), ZonedDateTime.now(ZoneId.of(userAnonymized.getTimeZoneId()))).toDays() + 1)
```

Comment:
Yes, I noticed it last night and made a fix, but didn't wait for the test results. These are positive, so I've pushed the update now.